examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -2593,7 +2593,9 @@ void sendMQTTDiscovery(const char *topic) {
 #endif  // MQTT_ENABLE
 
 void loop(void) {
+#if MDNS_ENABLE
   mdns.update();
```

Comment:
It appears `mdns.update()` is not needed/doesn't exist if we are on an ESP32.
Please add another conditional to exclude this line (and other locations) if on an ESP32.
Thanks.